.travis.yml

```diff
@@ -11,7 +11,7 @@ cache:
   directories:
     - node_modules
 
-script: yarn run jest --projects jest.*.config.js --maxWorkers=4
+script: yarn run jest --projects jest.*.config.js --maxWorkers=4 --reporters jest-silent-reporter
```

Comment:
Don't we have to add it as a dep? Otherwise it will, if at all, be only there through somebody else requiring it.

Reply:
@tdeekens it's a dep of `commercetools-frontend/jest-preset-mc-app` though

Reply:
I don't have too strong opinions then. I don't expect us to drop our preset but I often feel direct deps can't hurt.

Reply:
I'd vote for a direct dep as we use it directly.